source/Ldap.Tests/Ldap.Tests.csproj

```diff
@@ -7,22 +7,21 @@
     </PropertyGroup>
 
     <ItemGroup>
-        <PackageReference Include="Assent" Version="1.7.0" />
-        <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.7.1" />
+        <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.9.4" />
```

Comment:
Updated libraries in existing test project to match the newer versions the Integration Testing project came with by default